### ISSUE TITLE
ERROR (test-model.R:82:5): Graph attributes

### ISSUE THREAD

**Issue by mattdowle** [kkholst/lava#10]
Hi,
Thanks for using data.table. For the next release, in revdep testing I get the following error from R CMD check. It happens with data.table 1.13.2, too, as released on CRAN, so I don't think it is due to the data.table update. CRAN checks seem all OK for `lava` so it seems to be problem just local for me.  Any ideas please?  It would be nice to pass OK locally to clear this up.
I see that Suggested packages `gof` and `lava.tobit` have been removed from CRAN, but that doesn't seem to cause CRAN checks any problem. I'm running with _R_CHECK_FORCE_SUGGESTS_=false to get past that, which I assume CRAN checks do too.
Thanks, Matt

[00check.log](https://github.com/kkholst/lava/files/5639886/00check.log)
[test-all.Rout.fail](https://github.com/kkholst/lava/files/5647002/test-all.Rout.fail.txt)

**Comment by kkholst** [comment]
Could you share the output of sessionInfo() from you revdep testing environment?

**Comment by mattdowle** [comment]
```
> sessionInfo()
R version 4.0.3 (2020-10-10)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Pop!_OS 20.10

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3
LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.10.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices datasets  utils     methods   base     

loaded via a namespace (and not attached):
[1] compiler_4.0.3      BiocManager_1.30.10 tools_4.0.3        
> 
```

**Comment by mattdowle** [comment]
I uploaded the test-all.Rout.fail too, edited into the top comment.

**Comment by kkholst** [comment]
This was a problem in the unit test and indeed not a problem caused by data.table.  It should now be fixed in 22c2edbb6. As a temporary fix you can install Rgraphviz, `BiocManager::install("Rgraphviz")`, in which case the unit tests should also run through.

**Comment by mattdowle** [comment]
Great.
On `Rgraphviz` I already have it installed (`packageVersion("Rgraphviz")==2.34.0`). The `00check.log` would have mentioned `Rgraphviz` if it wasn't installed but it just mentions `gof` and `lava.tobit` (which are not available on CRAN or Bioc to install):
`Packages suggested but not available for checking: 'gof', 'lava.tobit'`.

**Comment by mattdowle** [comment]
I had another look and realized the error mentioned an R method not being found. I've seen reinstalling a package often fixes that so then I realized you probably meant reinstalling Rgraphviz above. So I tried to reinstall it but that failed and I was stuck for a while. Then I found https://github.com/kasperdanielhansen/Rgraphviz/issues/12 and indeed using `-O2` instead of `-O3` makes `Rgraphviz` install.
Now `lava` as on CRAN (v1.6.8.1 2020-11-04) is passing for me.
Thanks! All resolved!
